mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -556,11 +556,38 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 						"enable_confidential_compute": {
 							Type:        schema.TypeBool,
 							Required:    true,
+							ForceNew:    true,
 							Description: `Defines whether the instance should have confidential compute enabled.`,
 						},
 					},
 				},
 			},
+			"advanced_machine_features": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Computed:    true,
```

Comment:
Does this need to be Computed=true?

It seems like this would cause undesirable behavior.

Going from
```
advanced_machine_features {
  threads_per_core = 1
  enable_nested_virtualization = true
}
```
to
```
// removed
```
would NOT trigger an update/recreate. It would just be ignored, even tho the `threads_per_core` default value is 0.